### ISSUE TITLE
Crashses on firstVisiblePosition = ((LinearLayoutManager) getLayoutManager()).findFirstVisibleItemPosition()

### ISSUE THREAD

**Issue by technource-app** [Krupen/AutoplayVideos#28]
While plaplayAvailableVideos(0) called from onDraw(), it crashses app.

**Comment by Krupen** [comment]
Can you tell me from where are you calling `playAvailableVideos(0)` ?

**Comment by stefanborne** [comment]
I get this error also: 
`java.lang.NullPointerException: Attempt to invoke virtual method 'int android.support.v7.widget.LinearLayoutManager.findFirstVisibleItemPosition()' on a null object reference
                                                                                  at com.allattentionhere.autoplayvideos.AAH_CustomRecyclerView.playAvailableVideos(AAH_CustomRecyclerView.java:86)
                                                                                  at com.allattentionhere.autoplayvideos.AAH_CustomRecyclerView.onDraw(AAH_CustomRecyclerView.java:238)`

**Comment by Krupen** [comment]
Which version of support library are you using? And it would be really helpful if you can share code.

**Comment by stefanborne** [comment]
I use 25.3.1

`          recyclerView = (AAH_CustomRecyclerView) v.findViewById(R.id.messages);
            recyclerView.setActivity(getActivity());
            adapter = new FeedAdapterRecycler(getContext(), postsList, FeedNestedFragment.this);
            recyclerView.setAdapter(mAdapter);`

I'm settings the recyclerView from fragment. Do you need any other code? I set everything like you described.

**Comment by Krupen** [comment]
Can you try adding below code before calling `setAdapter(mAdapter)`?
```
   LinearLayoutManager mLayoutManager = new LinearLayoutManager(this);
   recyclerView.setLayoutManager(mLayoutManager);
```

**Comment by stefanborne** [comment]
Got the same error.

**Comment by Krupen** [comment]
I created another sample app that just uses this library with minimum features, but still not able to reproduce this error. Check the attached zip file. Also it would be great if you could share the project code in zip file getting this error.

https://drive.google.com/file/d/0Bys36lxF42L8RnBod0xoU2pKbWM/view?usp=sharing

**Comment by stefanborne** [comment]
Thanks, I'll check your app. It's not my personal project, so can't share it.

**Comment by Krupen** [comment]
Yes, I understand your concern, if you can create another sample app from your current project just reproducing this error, it would really help.

**Comment by technource-app** [comment]
I am getting this error on the very first time just assigning Id to Recyclerview in fragment.

I can share my code If you can provide me your email.

**Comment by Krupen** [comment]
okay, you can email me at 11bit009@niramuni.ac.in

**Comment by Krupen** [comment]
I just released an update version `0.1.4`. This version should fix the issue. 
If still not solved, please feel free to re-open the issue.
